### DISCUSSION TITLE
Welcome to pgsql-ogr-fdw Discussions!

### DISCUSSION THREAD

**Discussion by pramsey** [pramsey/pgsql-ogr-fdw#209] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jsdogbey** [comment]
Thanks Paul for the wonderful extension (ogr_fdw). I want to use it on mac os, please is there any guide on it. I have seen on Windows and Linux but nothing for OSX users can you help me Sir?
Thanks

**Comment by pramsey** [comment]
Sorry, nope. You'll need a development environment to build it, and enough background knowledge to do that, but I cannot offer tutoring or a step-by-step. Maybe lookup some "how to build software on macos" tutorials and start with a simpler extension like pgsql-http to get a feel for things.

**Reply by Jsdogbey** [reply]
Ok. Thanks so much for the swift response. I will go with your suggestion and start with "how to build software on macos". I am very grateful.
